tests/process_mining/test_pm4py_adapter.py

```
@@ -0,0 +1,142 @@
+import sys
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)